components/Typography/Text.tsx

```diff
@@ -70,6 +70,7 @@ export const Text = ({
         )
       }
     >
+      {children}
       {icon && (
         <HeroIcon
```

Comment:
I move the icon before the text in the Text component. The original one makes sense for My Account where we have icons (x and checkmarks) before error messages. For Campaign, we need the icon after the text (here for the "More" text after each chapter.